src/Entity/OgRole.php

```diff
@@ -243,6 +252,13 @@ public function delete() {
   }
 
   /**
+   * {@inheritdoc}
+   */
+  public function isDefaultRole() {
+    return in_array($this->getName(), [self::ANONYMOUS, self::AUTHENTICATED]);
```

Comment:
I changed it to `isRequired()` so it matches the other available methods such as `isNew()` and `isAdmin()`. I also updated the test coverage which was talking about 'default roles' where it should be 'required roles'.